### DISCUSSION TITLE
Solax integration blocks Home Assitant UI

### DISCUSSION THREAD

**Discussion by Zwer2k** [wills106/homeassistant-solax-modbus#593] [General]
I have noticed that the Solax integration keeps blocking the Home Assistant UI for 2-3 seconds. For example, switching actions are repeatedly executed with a delay or pages are sporadically loaded with a delay. As soon as I deactivate the integration, this no longer happens. It can be seen very clearly when scrolling faster between days in the energy dashboard.
Can anyone confirm this behavior?

**Comment by wills106** [comment]
Do you have any errors in your log? How are you connecting etc?

Sounds like you have a serious config error.

**Comment by Zwer2k** [comment]
No errors in the log. I use Waveshare RS485 TO ETH adapter. Home Assistant is installed as HAOS as Proxmox VM.
Inverter Sofar Solar HYD10KTL-3ph
Where could the configuration error be?
As I said. If the integration is deactivated, the problems are gone

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/10438794/97d99239-b0ae-459e-a0b5-31a65d88e4cc)

![image](https://github.com/wills106/homeassistant-solax-modbus/assets/10438794/cdb2ea3f-a837-4a55-a55c-a8decde8be1f)

**Comment by Zwer2k** [comment]
Unfortunately, the problem still exists.
